dev-cpp/tomlplusplus/tomlplusplus-3.4.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Header-only library for parsing  and serializing TOML"
+HOMEPAGE="https://github.com/marzer/tomlplusplus"
+SRC_URI="https://github.com/marzer/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
suggest ${P}.tar.gz -> ${P}.gh.tar.gz

Reply:
Having ${PN} in the  url is very redundant as well. makes renaming harder and doesnt add value there.

Similar rationale for why HOMEPAGE disallows it.